src/enums.ts

```diff
@@ -35,7 +35,6 @@ export enum IntegrationType {
 }
 
 export enum PackageManagerType {
-  Unknown = "Unknown",
```

Comment:
we should keep these, add the additional linux package managers and filter the options on the input within soos-sca